src/construction/construction.py

```diff
@@ -65,17 +74,23 @@ def run_construction(
     updated_snapshot_df = snapshot_df.copy()
 
     # Validate construction file and drop columns without constructed values
-    validate_data_with_schema(construction_df, "./config/construction_schema.toml")
+    validate_data_with_schema(construction_df, schema_path)
     construction_df = construction_df.dropna(axis="columns", how="all")
 
     # Add flags to indicate whether a row was constructed or should be imputed
     updated_snapshot_df["is_constructed"] = False
     updated_snapshot_df["force_imputation"] = False
     construction_df["is_constructed"] = True
 
-    # Add the years
-    updated_snapshot_df = create_period_year(updated_snapshot_df)
-    construction_df = create_period_year(construction_df)
+    # Create period_year column, except for NI which already has it
+    if not is_northern_ireland:
+        updated_snapshot_df = create_period_year(updated_snapshot_df)
+        construction_df = create_period_year(construction_df)
+
+    # Assign NI data a temporary dummy instance so the multi-index works
+    if is_northern_ireland:
+        updated_snapshot_df["instance"] = 0
+        construction_df["instance"] = 0
```

Comment:
Also here, Please could we make this "instance = 1"? this will help with the filtering we do for the outputs.

Reply:
The instance is only added temporarily so that `set_index()` works correctly, it's dropped immediately after so it doesn't appear in the final dataframe.

Reply:
Oh yes, sorry I didn't ready your code thoroughly. We do need the instance though, and it needs to be set as 1. I could update it in my branch... but then you'll need to change your code, so any chance you could set it to 1 here?

Reply:
Yep, I'll do that now!